.prettierrc

```
@@ -0,0 +1,8 @@
+{
+  "singleQuote": true,
+  "trailingComma": "all",
+  "printWidth": 100,
```

보통 80자가 기본값으로 알고 있는데 100으로 상호 협의가 된걸까요??

---

팀 내에서 큰 이유 없이 100자로 정하긴 했는데 보통 80자를 기본으로 사용하는군요
혹시 80자로 맞추는 게 더 나을까요?

---

어떤 설정이 좋다 나쁘다라기보다는 일반적으로 80자로 설정하는데 100자로 되어있어서 팀 내 협의가 된건지 여쭤본거였어요 ㅎㅎ 팀 내에서 동일한 코드 컨벤션을 가지는게 목적이기에 어떤 설정이든 근거있게 협의만 되면 상관없다고 봅니다
혹시 nest cli로 프로젝트 초기화할 때 80이 아니였나요??

---

```suggestion
{
  "singleQuote": true,
  "trailingComma": "all"
}
```
처음 초기화할 때 설정이 두 가지(singeQuote, trailingComma)만 있어서 미처 확인하지 못했네요.  
이전 미니 프로젝트에서 printWidth를 100으로 사용했었기 때문에, 메인 프로젝트에서도 그렇게 적용했습니다.
팀 내 협의된 기준이 있으니, 그럼 100으로 진행하겠습니다!